global_body_planner/include/global_body_planner/planning_utils.h

```diff
@@ -66,12 +65,48 @@ struct PlannerConfig {
       0.3;  // Duration of backup after finding an invalid state, s
   double BACKUP_RATIO =
       0.5;  // Ratio of trajectory to back up after finding an invalid state, s
-  int NUM_GEN_STATES =
+  int NUM_LEAP_SAMPLES =
       30;  // Number of actions computed for each extend function
   double GOAL_BOUNDS = 0.5;  // Distance threshold on reaching the goal (only
                              // used for vanilla RRT, not RRT-Connect)
   double MAX_TIME = 2.0;     // Maximum planning time allowed
   Eigen::Vector3d G_VEC;     // Maximum planning time allowed
+
+  // Define robot params and declare points used for validity checking
+  double ROBOT_L = 0.4;  // Length of robot body, m (0.6 cheetah, 0.554 ANYmal)
+  double ROBOT_W = 0.3;  // Width of robot body, m (0.256 cheetah, 0.232 ANYmal)
+  double ROBOT_H = 0.05;  // Vertical distance between leg base and bottom of
+                          // robot, m (0.1 cheetah, 0.04 ANYmal)
+
+  double body_traversability_threshold = 0.01;
```

Comment:
Should these come from constructor or param server?

Reply:
They get loaded from the param server, these are just defaults (although they are used for the tests. Would we prefer to keep it that way it have the tests load from the server?

Reply:
I think have the tests load from server as thats how we have it now elsewhere

Reply:
Okay, added a function to the PlannerConfig struct (could make this its own class in a future commit) which loads everything from the server, removed the default arguments, and implemented it within the test fixture and the ros interface.